src/bot_commands.py

```diff
@@ -1,104 +1,109 @@
-        raise NotFoundError(f"Birthday date is unknown for {name}")
-    
-    return f"Contact name: {record.name.value}, birthday: {record.birthday.value.strftime("%d.%m.%Y")}"
-
-
-@show_all_error
-#функцію і метод потрібно доробити із врахуванням вхідного аргументу кількості днів, по замовченню нехай буде 7
-def birthdays(book: AddressBook):
-    if not book:
-        raise ValueError
-    
-    upcoming_birthdays = book.get_upcoming_birthdays()
-
-    if not upcoming_birthdays:
-        return "No upcoming birthdays in the next 7 days."
-
-    return "\n".join(f"Contact name: {data["name"]}, birthday: {data["birthday"]}, congratulation date: {data["congratulation_date"]}" for data in upcoming_birthdays)
+from src.handlers import (
+    add_contact,
+    add_phone,
+    add_address,
+    add_email,
+    change_phone,
+    change_birthday,
+    change_email,
+    change_address,
+    delete_contact,
+    delete_phone,
+    show_all,
+    search_by_phone,
+    search_by_email,
+    show_phone,
+    add_birthday,
+    show_birthday,
+    birthdays,
+)
+
+help_commands = {
+    "help": {
+        # add handler that will show menu or description
+        "handler": lambda *args: "How can I help you?",
+        "description": "shows all menu options or a specific comand description",
+    },
+}
 
-#для повноцінного меню необхідно розширити словник команд, а саме додати опис кожної команди
-commands = {
-    "hello": lambda *args: "How can I help you?",
-    "add": lambda args, book: add_contact(args, book),
-    "change": lambda args, book: change_contact(args, book),
-    "phone": lambda args, book: show_phone(args, book),
-    "all": lambda args, book: show_all(book),
-    "add-birthday": lambda args, book: add_birthday(args, book),
-    "show-birthday": lambda args, book: show_birthday(args, book),
-    "birthdays": lambda args, book: birthdays(book),
+exit_commands = {
+    "close": {
+        "description": "exit the application",
+    },
+    "exit": {
+        "description": "exit the application",
+    },
 }
 
-exit_commands = ["close", "exit"]
\ No newline at end of file
+contact_commands = {
+    "add-contact": {
+        "handler": lambda args, book: add_contact(args, book),
+        "description": "adds a new contact, requires 'name' and 'phone number'",
+    },
+    "add-phone": {
+        "handler": lambda args, book: add_phone(args, book),
+        "description": "adds additional phone number to the specified contact, requires 'name' and 'phone number'",
+    },
+    "add-birthday": {
+        "handler": lambda args, book: add_birthday(args, book),
+        "description": "adds birthday date to the specified contact, requires 'name' and 'birthday date'",
+    },
+    "add-email": {
+        "handler": lambda args, book: add_email(args, book),
+        "description": "adds email to the specified contact, requires 'name' and 'email'",
+    },
+    "add-address": {
+        "handler": lambda args, book: add_address(args, book),
```

Comment:
Мені здається lamda тут лишня, достатньо просто (маю на увазі усі хендлери)
```suggestion
        "handler": add_address
```

Reply:
good suggestion, thanks )